wsmaster/che-core-api-auth/src/main/java/org/eclipse/che/security/oauth1/OAuthAuthenticationService.java

```diff
@@ -79,12 +79,10 @@ public Response callback() throws OAuthAuthenticationException, BadRequestExcept
 
     try {
       getAuthenticator(providerName).callback(requestUrl);
+    } catch (UserDeniedOAuthAuthenticationException e) {
+      redirectUriBuilder.queryParam(ERROR_QUERY_NAME, "access_denied");
```

Comment:
Maybe `user_denied` is better. That tells explicitly that the user declined the authorization request. WDYT?

Reply:
Yea sounds good. access_denied is from OAuth2 but since we cannot use OAuth2 error query parameter with its values we can use this user_denied value which is more readable.

Reply:
I'm OK with any but please keep us informed about changes.
The work on Dashboard is in progress already

Reply:
let's keep `access_denied` . The same as it is on OAuth2